Aesthetic Changes

AsianHam opened the pull request:
Limited tasks within a div with a scrollbar to prevent "Add Task" button from shifting

Changed delete button icon to trash icon

Changed deleted button icon to red color

AsianHam committed:
Scrollbar

Set height for div containing tasks

"Add Task" button no longer moves with length of list

AsianHam committed:
Updated Delete button icon

Changed delete button to a trashcan icon

Changed delete button to a red color

Jacksondr5 reviewed (changes requested):
It’s probably better to npm install the antd icon library instead of adding the images directly

Jacksondr5 commented:
> It’s probably better to npm install the antd icon library instead of adding the images directly

Oh wait you did that.  Why do you also have the trash can images then?

AsianHam committed:
Removed extraneous fiiles

deleted two images that were no longer needed

Jacksondr5 commented on todo/src/App.js:
```diff
@@ -44,27 +44,28 @@ export default function App() {
+                </Typography.Text>
+              </Checkbox>
+              <div className="delButton" key={index}>
+                <Button icon={<DeleteOutlined/>} danger onClick={(e) => deleteTask(e, index)}></Button>
```
The danger property is supposed to make this red right?  Doesnt look like its doing that

AsianHam replied:
It makes the delete button red, which it is doing

jackson-miller-tbol replied:
Yeah my bad I was looking at the wrong branch when I was testing this

Jacksondr5 reviewed (changes requested):
![image](https://user-images.githubusercontent.com/9014034/163678476-7c20732c-ba9a-423b-a9c0-d13821b202ee.png)
You should find some way to force the text to flow to the next line so that doesnt happen.  Then make sure the delete button is centered vertically when the text flows

Jacksondr5 commented:
I would consider moving the edit icon to the left of the text, see if it looks better to have uniform edit icons or not.

AsianHam committed:
Text Overflow Fix

Restructured checkbox and delete divs to be floating divs

Limited checkbox div size to keep delete button in place

Text will overflow downward

AsianHam commented:
> I would consider moving the edit icon to the left of the text, see if it looks better to have uniform edit icons or not.

I can't move the edit icon due to how Typography.Text works. I could replace the checkbox text with an editable input field if we really wanted to.

AsianHam committed:
Input Update

Swapped Typography for TextArea Input

Changed div positioning using CSS Grids instead of Float

Added blur functionality on Enter Key press

Added focus functionality when creating a new task using observer

Loss of strikethrough functionality

Code cleanup
